src/packagedcode/npm.py

```diff
@@ -339,22 +407,20 @@ def split_scoped_package_name(name):
     return ns, name
 
 
-def get_licensing(license_object):
+def get_declared_licenses(license_object):
     """
-    Return actual declared licensing info from a license_data object that can be
-    a string, list, dict or dict list and come from the license or licenses
-    package.json tag.
+    Return the declared licenses, which is a list of order dictionary of licenses
     """
     if not license_object:
-        return
+        return []
```

Comment:
We need to be clear about what is the declared license of an npm. IMHO we can have all these cases:
we can have both "license" and "licenses"
- `license` should only be a single string when present. But we still need to test if this is a list of strings, dict or list of dicts.
- `licenses` can be a list of strings, a dict or a list of dict.

And we can have weird cases we have both `license` and `licenses`


So we could:
1. always return a list of things that contains either strings or dicts
2. return a string if we have a single `license` and no `licenses` (default common cases) or a list otherwise as in 1.

Below we are implementing 2.  but above we are returning an empty list which would be more consistent with 1. None would be better  for 2.

But after all I think it would be simpler in this function to always return a list.
So the line below e.g. "return license_object" should be changed to `return [license_object]`